cli/Cargo.toml

```diff
@@ -36,10 +36,12 @@ regex = "1.1.6"
 remove_dir_all = "0.5.1"
 ring = "0.14.6"
 rustyline = "4.0.0"
+serde = "1.0.91"
+serde_derive = "1.0.91"
 serde_json = "1.0.39"
 source-map-mappings = "0.5.0"
 tempfile = "3.0.7"
-tokio = "0.1.19"
+tokio = "0.1.20"
```

Comment:
When you land this, be sure to separate the "crate upgrade" commit from prog-macro additions.